code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -158,3 +158,15 @@
 	desc = "Used to help with blood loss."
 	icon_state = "pill2"
 	list_reagents = list("iron" = 10, "salglu_solution" = 10)
+
+/obj/item/reagent_containers/food/pill/strange_reagent
+	name = "\improper Strange Reagente pill"
```

Comment:
```suggestion
	name = "\improper Strange Reagent pill"
```

little typo here, you may want to wait with this until #20614 gets merged or closed (and then edit its description if you want it to match with the reagent's)

Reply:
I will keep my eye open for that PR. Thank you for the heads up!